### ISSUE TITLE
Multiple team members speed inputs to reduce speed rng range

### ISSUE THREAD

**Issue by zx900930** [Apen/epicseven#14]
Some interesting feedback here:
Add another 2 teammates speed and CR% datas(optional)
to get another 2 speed rng ranges.
Then we calculate the intersection of these 3 intervals.
Which will reduce the overall speed rng range.

**Comment by Apen** [comment]
Yes it can be interesting but it will complicate the user experience.
Imo, this need to an "hide/show" feature only for advanced users.

**Comment by zx900930** [comment]
Yes, add a collapse menu for it.
Also the enemies heros like F.Lidica has a self push 75% and team push 15%.
In our current page, I have to put 15% as the CR push value, then manually -60% in F.Lidica's CR%.
Do we need to detect the hero and automatically load the CR push data?
